.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: gh-pages
+on:
+  release:
+    types: [published, edited]
+
+concurrency:
+  group: gh-pages
+
+jobs:
+  update-helm-repo:
+    name: Update gh-pages helm repo index
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install helm
+      run: |
+        curl -sfL https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash -s -- --version v3.12.3
+
+    - name: Check out repo
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Update repo index
+      run: |
+        git config user.name "Github Actions"
+        git config user.email "no-reply@github.com"
+        ./scripts/github/update-helm-repo.sh ${{ join(github.event.release.assets.*.browser_download_url, ' ') }}
+
+    - name: Push
+      run: git push -f https://${GITHUB_ACTOR}:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }} gh-pages
```

Comment:
no strong reasoning, but i found using `stefanzweifel/git-auto-commit-action@v4` easier and cleaner to push to remote. But as said, I'm not against having plain git commands on the workflow, just heads up.

Reply:
I'd like to only rely on "official" actions or at least actions from verified publishers to voting against stefanzweifel. It would look slightly cleaner but doesn't even save any lines of yaml